### DISCUSSION TITLE
VS screen select screen enhancements

### DISCUSSION THREAD

**Discussion by yoxcheatdeath** [ikemen-engine/Ikemen-GO#1861] [Feature Request] [answered]
### 

currently we can have face1 and face2 for each member in select screen but not for vs screen
we also do not have the capability to individually offset names for each member on vs screen

### Describe the solution you'd like

for the first portion it's adding pX.face1.num=4 and pX.face2.num=4 compatibility with individual .memberX options

and pX.memberX.name options including individual .memberX options

### Additional context

![ikemen005](https://github.com/ikemen-engine/Ikemen-GO/assets/174139247/64834a20-ecaa-4443-b91f-4d51451cf914)

seen here, P1/P2 are widow and doom, p3/p4 are upper corners morrigan and alex, p5/p6 are lower corners ruby hear and batsu. 
currently we aren't able to individually place the names so they aren't capable of following more complex portrait placements

### Search terms

vs screen

**Comment by K4thos** [comment, marked as answer]
all of this is possible in nightly
